pom.xml

```diff
@@ -53,7 +53,7 @@
 		<!-- other third party libraries versions -->
 		<slf4j.version>1.7.7</slf4j.version>
 		<logback.version>1.0.13</logback.version>
-		<sesame.version>2.7.13</sesame.version>
+		<sesame.version>2.8.1</sesame.version>
```

Comment:
To supplement the comment: The idea is that sesame.version should be updated in uv-pom (https://github.com/UnifiedViews/Plugin-DevEnv/blob/master/uv-pom/pom.xml) and Core/pom.xml then reuses such version (uv-pom is parent of Core/pom). 

sesame.version should be removed from Core/pom.xml.
Similarly logback.version and slf4j.version should be taken from uv-pom/pom.xml